src/Token/AccessTokenInterface.php

```diff
@@ -69,4 +70,20 @@ public function __toString();
      * @return array
      */
     public function jsonSerialize();
+
+    /**
+     * Sets the clock.
+     *
+     * @param Clock $clock a clock.
+     *
+     * @return void
+     */
+    public function setClock(Clock $clock);
+
+    /**
+     * Get the current time, whether real or simulated.
+     *
+     * @return int
+     */
+    public function getTimeNow();
```

Comment:
If you choose to add `getClock(): Clock`, and the Clock has a `now(): DateTimeInterface` method, then is there any reason to have `getTimeNow()`?

Reply:
I dont see how both `getTimeNow` and a theoretical `public getClock()` method would be useful for access tokens. See also my note about whether you'd prefer for the previously implemented time functionality (`getTimeNow`/`setTimeNow`/`resetTimeNow`/`\League\OAuth2\Client\Token\AccessToken::$timeNow`) to be removed.

Reply:
If you add a `getClock()` method, then I agree that `getTimeNow()` is no longer necessary.